### DISCUSSION TITLE
Combine meshes that use the same material

### DISCUSSION THREAD

**Discussion by rawnsley** [donmccurdy/glTF-Transform#406] [Ideas]
Does glTF-Transform have an optimization function to combine meshes that use the same material? I've seen this technique used successfully elsewhere to reduce the number of draw calls.

**Comment by donmccurdy** [comment]
Not currently, although this is something I've wanted to include for a while – there's an issue tracking it, and a bit of discussion, in https://github.com/donmccurdy/glTF-Transform/issues/11.
